packages/aws-cdk/lib/init.ts

```diff
@@ -194,12 +193,20 @@ interface ProjectInfo {
   readonly name: string;
 }
 
+function versionedTemplatesDir(): Promise<string> {
+  return new Promise(async resolve => {
+    const majorVersion = versionNumber().replace(/\..+/, '');
```

Comment:
can we instead parse this using [`semver`](https://www.npmjs.com/package/semver)?